content/attacks/posts/2023-11-23-HecoBridge.md

```diff
@@ -0,0 +1,61 @@
+---
+date: 2023-11-23
+target-entities: 
+   - Heco Bridge
+   - HTX
+entity-types:  
+   - Custodian
+   - DeFi
+   - Exchange
+   - Bridge
+attack-types:
+   - Private Key Leak
+title: Coordinated Attacks Result in $113.3 Million in Losses for Heco Bridge and HTX Exchange
+loss: 113300000
+---
+
+## Summary
+
+On November 22, 2023, Heco Bridge and HTX Exchange were victims of cyberattacks, resulting in over $113.3 million in losses. The attacks appear coordinated and carried out by the same attacker based on similar exploitative techniques and the connection between the two targets. Blockchain security firms CertiK, Peckshield, and Cyvers have reported over $86.6 million in digital assets losses for Heco Bridge and $13.6 million in losses for HTX. All of which were distributed over eight wallets. Additionally, CertiK noted several transactions taking place simultaneously on TRON, totaling $12.6 million, putting their reported total over $113.3 million.
+
+## Attackers
+
+The attacker has yet to be identified but used the [following wallet](https://etherscan.io/address/0xe47e6dA16Bb83EB0FD26b3F29b15CE8Fab089B9e) to transfer some of the drained funds:
+
+        - 0xe47e6dA16Bb83EB0FD26b3F29b15CE8Fab089B9e
+        
+## Losses
+
+According to CertiK, the following [withdraws](https://www.certik.com/resources/blog/39YOzflgCCbfI9evJliCeQ-heco-bridge-exploit) took place: 
+
+Heco Bridge:
+   - $10,145 in ETH
+   - $42,110,000 in USDT
+   - $489 in HBTC
+   - $346,867,120,000 in SHIBA INU
+   - $173,200 in UNI
+   - $619,000 in USDC
+   - $42,399 in LINK
+   - $346,994 in TUSD
+
+HTX:
+   - $1,240 in ETH
+   - $7,330,600 in USDT
+   - $1,780,000 in USDC
+   - $61,250 in LINK
+   - $2,195,836 in ARIX
+   - $4,254,541 in KOK
+
+## Timeline
+
+   - **November 22, 2023, 10:06 AM UTC:** Initial [funds transfer](https://etherscan.io/tx/0xe021e1d8fd38a874de4713b7ce1aaffd646a135265826f6eb3232e05313b2d87) begins.
+   - **November 22, 2023 at 10:56 AM UTC** Heco Bridge begins the process of moving recovery funds into [wallet.](https://etherscan.io/address/0x18709e89bd403f470088abdacebe86cc60dda12e)
```

Comment:
It's different hacking case, happened on October

Reply:
Looks like the wallet was created for the October hack, but more recovery funds were moved into it for the November hack. Do you want me to add the transaction hash or delete the line?

Reply:
The hash is wrong, I guess the line should be deleted.